jekyll/_cci2/api-intro.md

```diff
@@ -37,13 +37,20 @@ Currently, [Personal API tokens]({{site.baseurl}}/managing-api-tokens/#creating-
 ## Getting started with the API v2
 {: #getting-started-with-the-api-v2 }
 
+**GitLab SaaS Support users:** Note that the definition of **project slug** provided in this section, as well as its usage throughout this document applies to GitHub and Bitbucket projects only. GitLab projects currently use a new slug format:  
+<br>
+`circleci/:org-short-id/:project-short-id`  
```

Comment:
@zalimeni @nathanfishcircleci I was wondering if you might comment specifically on using "org short ID" and "project short ID" here. I didn't think we wanted to say analytics ID, and I couldn't think of any better terminology / way to describe the string.

Reply:
Thanks for the callout here @kbatuigas . I agree analytics ID would not be desired, since these are opaque strings - technically, even the `/` is meant to be opaque here and never split, so I'm inclined to say something like

`circleci/:slug-remainder` or 
`circleci/:unique-slug-part`

and not even try to explain the project vs. org `/`. Kat and @nathanfishcircleci does that sound reasonable to you?

Otherwise I think this change looks good, and really appreciate y'all's work on it.

Reply:
Hmm. I just realized this would probably create confusion because folks would not have clear instruction as to what segment of the URI to copy.

It'd be nice if we exposed the slug as a copy-able value in the UI, like UUIDs in org/project settings, so we could just point folks there instead.

I'd defer to Fish on the current docs change; I suppose we could iterate to better "hide" the `/` from users by providing a less manual process for copying it from their browser.